src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -3,41 +3,48 @@ import { arrayOf, shape, number, oneOf, oneOfType, string } from 'prop-types';
 import pathOr from 'ramda/src/pathOr';
 import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
 
+import isLive from '#lib/utilities/isLive';
 import { storyItem, linkPromo } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import { RequestContext } from '#contexts/RequestContext';
 import CpsOnwardJourney from '../CpsOnwardJourney';
 import StoryPromo from '../StoryPromo';
 import FrostedGlassPromo from '../../components/FrostedGlassPromo/lazy';
 import useViewTracker from '#hooks/useViewTracker';
-import useToggle from '#hooks/useToggle';
+import useOptimizelyVariation from '#hooks/useOptimizelyVariation';
 
 const eventTrackingData = {
   block: {
     componentName: 'features',
   },
 };
 
+const HIGH_IMPACT_VARIATION = 'variation_1';
+
 const PromoListComponent = ({ promoItems, dir }) => {
   const { serviceDatetimeLocale } = useContext(ServiceContext);
   const viewRef = useViewTracker(eventTrackingData.block);
   const { isAmp } = useContext(RequestContext);
-  const { enabled: frostedPromoEnabled, value: frostedPromoCount } =
-    useToggle('frostedPromo');
 
-  const selectComponent = index => {
-    if (isAmp) return StoryPromo;
-    if (!frostedPromoEnabled) return StoryPromo;
-    return index + 1 <= frostedPromoCount ? FrostedGlassPromo : StoryPromo;
-  };
+  const promoVariation = useOptimizelyVariation(
+    'high_impact_feature_analysis_promo',
```

Comment:
this string should maybe be a constant as it's used in 2 places and at risk of being changed here and not the other place

Reply:
Done